libretroshare/src/rsserver/rsinit.cc

```diff
@@ -22,6 +22,8 @@
 /// RetroShare initialization and login API implementation
 
 #include <unistd.h>
+#include <math.h>
```

Comment:
Should us `<cmath>` instead of `<math.h>`

Reply:
done